server/webapp/WEB-INF/rails.new/app/assets/javascripts/build_output_observer.js

```diff
@@ -72,15 +72,21 @@ BuildOutputObserver.prototype = {
 
     _update_live_output_raw: function (build_output) {
         var is_output_empty = !build_output;
-        var buildoutputPreElement = $('buildoutput_pre');
+        // we loop through these elements below, because there's 2 of those
+        // on on the console tab, and another on failures tab
+        var buildoutputPreElement = $$('.buildoutput_pre');
         if (!is_output_empty && buildoutputPreElement) {
             var escapedOutPut = build_output.escapeHTML();
             if (Prototype.Browser.IE) {
                 // Fix for the IE not wrap /r in pre bug
                 escapedOutPut = '<br/>' + escapedOutPut.replace(/\n/ig, '<br\/>');
-                buildoutputPreElement.innerHTML += escapedOutPut;
+                buildoutputPreElement.each(function(e, i){
+                    e.innerHTML += escapedOutPut;
```

Comment:
When used like this might want to consider the append() or the html() method?
http://stackoverflow.com/questions/18393981/append-vs-html-vs-innerhtml-performance 

Also jquery takes care of multiple element matches right?

```
if (Prototype.Browser.IE) {
    // Fix for the IE not wrap /r in pre bug
    escapedOutPut = '<br/>' + escapedOutPut.replace(/\n/ig, '<br\/>');
    buildoutputPreElement.append(escapedOutPut);
} else {
    buildoutputPreElement.insert({bottom: escapedOutPut});
}

```
Works for me